src/SourceGenerators/Uno.UI.SourceGenerators.Internal/TSBindings/TSBindingsGenerator.cs

```diff
@@ -18,9 +18,10 @@ class TSBindingsGenerator : ISourceGenerator
 		private string _bindingsPaths;
 		private string[] _sourceAssemblies;
 
-		private static INamedTypeSymbol _intPtrSymbol;
-		private static INamedTypeSymbol _structLayoutSymbol;
-		private static INamedTypeSymbol _interopMessageSymbol;
+		private INamedTypeSymbol _intPtrSymbol;
+		private INamedTypeSymbol _uintPtrSymbol;
+		private INamedTypeSymbol _structLayoutSymbol;
+		private INamedTypeSymbol _interopMessageSymbol;
```

Comment:
The compiler is free to re-use generator instances:

https://learn.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.iincrementalgenerator?view=roslyn-dotnet-4.6.0

> The lifetime of a generator is controlled by the compiler. State should not be stored directly on the generator, as there is no guarantee that the same instance will be used on a subsequent generation pass.

I think this might still be problematic. It's probably best to pass these around.

For IntPtr, there is actually a SpecialType for it, so we might not need that symbol at all.